### PR TITLE
hello

### PR DESCRIPTION
**Pull request by ywen407**
hello file added

### CODE REVIEW

**Review by Yangeok — COMMENTED** [review]
ㅎㅎ..

**Review by Yangeok — CHANGES_REQUESTED** [review]
ㅎㅎㅎ

**Review by Yangeok — APPROVED** [review]
ㅎㅎㅎㅎ

File: hello.js [hunk 1]
```diff
@@ -0,0 +1 @@
+var a =4;
```
Comment (Yangeok) [inline hello.js:1]: ㅇㅈㅂㅇㅂㅈㅇ

**Review by Yangeok — APPROVED** [review]

File: hello.js (hunk 1 above)
Comment (ywen407) [reply hello.js:1]: ??

**Pull request merged**